spark/src/main/java/org/apache/spark/shuffle/comet/CometTestShuffleMemoryAllocator.java

```diff
@@ -67,11 +68,10 @@ public final class CometTestShuffleMemoryAllocator extends CometShuffleMemoryAll
   private static final int OFFSET_BITS = 51;
   private static final long MASK_LONG_LOWER_51_BITS = 0x7FFFFFFFFFFFFL;
 
-  private static CometTestShuffleMemoryAllocator INSTANCE;
```

Comment:
The `isSparkTesting` variable in the other file made me glance at the unused warnings in the files I was touching, and this isn't used.